library/kotlin/src/io/envoyproxy/envoymobile/Client.kt

```diff
@@ -0,0 +1,73 @@
+package io.envoyproxy.envoymobile
+
+import java.nio.ByteBuffer
+
+interface Client {
+  fun startStream(request: Request, streamCallback: StreamCallback): StreamEmitter
```

Comment:
Interesting that this function returns a `StreamEmitter`. I think this makes sense assuming it's more or less to be used as the platform equivalent of a `envoy_stream_t` defined [here](https://github.com/lyft/envoy-mobile/blob/ae3daf9f788ee863a146424042c388c6720efc52/library/common/main_interface.h#L18)?

Reply:
Yeah, I was trying to follow exactly what the common layer exposes with very little modifications